Library/Homebrew/upgrade.rb

```diff
@@ -170,7 +170,7 @@ def check_installed_dependents(args:)
       oh1 "Checking for dependents of upgraded formulae..." unless args.dry_run?
       broken_dependents = CacheStoreDatabase.use(:linkage) do |db|
         installed_formulae.flat_map(&:runtime_installed_formula_dependents)
-                          .select do |f|
+                          .uniq.select do |f|
```

Comment:
```suggestion
                          .uniq
                          .select do |f|
```

Reply:
sounds good; I'll apply this manually and squash

Reply:
12070b4